cpg_workflows/jobs/vcf.py

```diff
@@ -57,6 +57,38 @@ def subset_vcf(
     return j
 
 
+def quick_and_easy_bcftools_concat(
+    input_vcfs: list[hb.ResourceGroup | hb.Resource],
+    storage_gb: int = 10,
+    job_attrs: dict | None = None,
+) -> Job:
+    """
+    A quick and easy way to concatenate VCFs
+
+    Args:
+        input_vcfs (list[hb.ResourceGroup]): list of VCFs to concatenate, requires the 'vcf.gz' attribute
+        storage_gb (int): storage size for the job
+        job_attrs (dict): job attributes, or None
+
+    Returns:
+        a ResourceGroup with the concatenated VCF, with the 'vcf.gz', and 'vcf.gz.tbi' attributes
+    """
+    j = get_batch().new_job(f'Concat {len(input_vcfs)} VCFs', (job_attrs or {}) | {'tool': 'bcftools concat'})
+    j.image(image_path('bcftools'))
+    res = STANDARD.set_resources(j, storage_gb=storage_gb)
+
+    # declare a resource group for the concatenated output
+    j.declare_resource_group(output={'vcf.gz': '{root}.vcf.gz', 'vcf.gz.tbi': '{root}.vcf.gz.tbi'})
+    j.command(
+        f"""
+    bcftools concat --threads {res.get_nthreads() -1 } -a {" ".join(vcf["vcf.gz"] for vcf in input_vcfs)} \
+    -Oz -o {j.output['vcf.gz']}
+    tabix -p vcf {j.output['vcf.gz']}
```

Comment:
It's probably better future compatibility to use bcftools index (to make `csi` indexes) over tabix, unless you really want `tbi` indexes for some reason

Reply:
mmm, fair point. I'll edit this to make both. Hail can export VCFs but only with `tbi`s, so I've just tried to keep up with that throughout.